docs/var.tfvars-doc.md

```diff
@@ -319,6 +319,16 @@ rhcos_kernel_options        = []
   rhcos_kernel_options      = ["slub_max_order=0","loglevel=7"]
   ```
 
+This is a Map of [Node labels](https://kubernetes.io/docs/reference/labels-annotations-taints) and its values. Some of the well known labels such as region, zone and instance-type are automated. More custom labels can be added using the `node_labels` map variable.
```

Comment:
It is better to mention exact map keys instead of generic `region, zone and instance-type`.